src/components/todo/index.js

```diff
@@ -23,17 +26,30 @@ class TodoPage extends Component {
     return (
       <Paper className={s.container}>
         {/* {this.renderDelete()} */}
-        <h1>To do List</h1>
+        <DeleteDialog open={this.state.modal} title={this.state.title} content={ `Anda akan menghapus ${this.state.title}`} onClose={() => {
```

Comment:
for styling, we normally will try to avoid the code to get too long in a single line.
better to style it like this:
```
<DeleteDialog 
  open={this.state.modal} 
  title={this.state.title} 
  content=...
/>
```
you can use VSCode eslint plugin to detect this